Update script import_vcf - import Mastermind variants

dglemos opened the pull request:
Update import_vcf.pl to import variants from Mastermind vcf files. 
This update is needed to import the variants into private db.

ima23 reviewed (commented):
Overall looks good and this script works as expected. Was used for e104 production. Minor notes on:
- can a few comments be added please
- if chr_synonyms is required for source Mastermind, can a check for this parameter setting be added?
I recommend a ticket be created to review the rest of the script the help documentation is not up to date (link not current), registry should be listed as required (otherwise fails), a systematic review of the indentation would also be useful.

dglemos commented on scripts/import/import_vcf.pl:
```diff
@@ -774,13 +786,32 @@ sub main {
+      if($config->{source} eq 'Mastermind') {
+        $var_name = $data->{info}->{HGVSG};
+        if($var_name =~ /,/) {
+          $var_name =~ s/.*,//;
```
Added a comment.

dglemos commented on scripts/import/import_vcf.pl:
```diff
@@ -2788,6 +2823,31 @@ sub print_file{
+
+  my $file = $config->{chr_synonyms};
+
+  open(my $fh, '<:encoding(UTF-8)', $file)
```
No need to have the encoding.

dglemos commented:
@ima23 thank you for the review! I addressed your comments, let me know if I missed something. About the script review, I'm going to open a ticket for future review.

ima23 reviewed (approved):
Thanks for the updates. Looks good to merge.

ima23 commented:
merged to `master` and `release/104`
